src/containers/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import Weather from '../../components/Weather';
+
+export class Dashboard extends Component {
```

Comment:
You are exporting as default this class at the end of the file, export here is not necessary (same for all the other modules if you export default at the end).